matchbox_socket/src/webrtc_socket/native/signalling_loop.rs

```diff
@@ -8,14 +8,31 @@ use futures_util::select;
 use log::{debug, error, warn};
 
 pub async fn signalling_loop(
+    mut attempts: u16,
```

Comment:
Wouldn't it make sense to make this an `Option`, so you have the option of just retrying infinitely?

Reply:
Yes, it would. It was at this point I was just throwing anything at the wall and seeing what stuck after the prior attempts. I will make sure to change that. ;)